src/components/layout/PanelChange/styles.module.scss

```diff
@@ -3,8 +3,8 @@
   @include flexbox(flex-start, center);
   flex-direction: column;
   border-radius: 10px;
-  background: var(--color-blue-light);
```

Comment:
its always better to give variable colors and then use it, rather than giving constant value. 🙏🏼